makefile

```diff
@@ -9,22 +9,22 @@ osm:
 	docker-compose exec workspace osm-initial-import $(ARGS) -s -H postgres-postgis
 
 run_kartotherian:
-	docker-compose exec kartotherian bash -c ". /.nvm/nvm.sh && nvm use 10.15.2 && nodemon --ext js,json,yaml --signal SIGHUP server.js -c config.docker.yaml"
+	docker-compose exec kartotherian nodemon --ext js,json,yaml --signal SIGHUP server.js -c config.docker.yaml
 
 npm_test:
-	docker-compose exec kartotherian bash -c ". /.nvm/nvm.sh && nvm use 10.15.2 && npm test"
+	docker-compose exec kartotherian npm test
 
 npm_install:
-	docker-compose exec kartotherian bash -c ". /.nvm/nvm.sh && nvm use 10.15.2 && npm install --unsafe-perm"
+	docker-compose exec kartotherian npm install --unsafe-perm
 
 npm_link:
-	docker-compose exec kartotherian bash -c "cd /srv/dependencies/osm-bright.tm2source && . /.nvm/nvm.sh && nvm use 10.15.2 && npm link"
-	docker-compose exec kartotherian bash -c "cd /srv/dependencies/osm-bright.tm2 && . /.nvm/nvm.sh && nvm use 10.15.2 && npm link"
-	docker-compose exec kartotherian bash -c "cd /home/kartotherian/packages/kartotherian && . /.nvm/nvm.sh && nvm use 10.15.2 && npm link @kartotherian/osm-bright-source"
-	docker-compose exec kartotherian bash -c "cd /home/kartotherian/packages/kartotherian && . /.nvm/nvm.sh && nvm use 10.15.2 && npm link @kartotherian/osm-bright-style"
+	docker-compose exec kartotherian -w /srv/dependencies/osm-bright.tm2source npm link
```

Comment:
Just curious, what is `-w` for?

Reply:
This is a flag for docker-compose to set the working directory before running a command, so replaces the "cd" in the previous code.  I probably should have chosen the more verbose option `--workdir`, https://docs.docker.com/engine/reference/commandline/compose_exec/#options